bot/src/main/java/com/community/tools/service/NotificationService.java

```diff
@@ -0,0 +1,121 @@
+package com.community.tools.service;
+
+import com.community.tools.discord.DiscordService;
+import com.community.tools.dto.UserForTaskStatusDto;
+import com.community.tools.model.TaskNameAndStatus;
+import com.community.tools.service.github.DiscordGithubService;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import net.dv8tion.jda.api.entities.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class NotificationService {
+
+  private static final String NOT_FOUND_DISCORD_USER_MESSAGE = "User with this discord name"
+      + " not found";
+
+  @Value("${text.channel}")
+  private String textChannelName;
+  @Autowired
```

Comment:
field injection is not recommended, better to use constructor

Reply:
Agreed, but I was guided by an approach that already used in project